### DISCUSSION TITLE
Problem running samba 4.15.1

### DISCUSSION THREAD

**Discussion by grigutis** [Homebrew/discussions#2509] [Everyday usage]
I just installed samba 4.15.1 and have a valid smb.conf, but it exits with a non-zero exit code right after I start it.
```
% sudo /usr/local/sbin/samba-dot-org-smbd -i
smbd version 4.15.1 started.
Copyright Andrew Tridgell and the Samba Team 1992-2021
% echo $?
1
```
Since it exits right after it starts I'm not seeing any logs anywhere.
Has anyone else run into this or have any suggestions?

**Comment by undulattice** [comment]
Did you manage to fix this?
Encountering the same thing...

**Reply by grigutis** [reply]
No, I gave up.
